lib/str.js

```diff
@@ -943,8 +943,14 @@ const Str = {
      * @returns {bool}
      */
     isValidMention(mention) {
-        // A valid mention starts with a space, *, _, #, ', ", or @ (with no preceding characters).
-        return /[\s*~_#'"@]/g.test(mention.charAt(0));
+        // A valid mention starts with a space or @, or
+        // starts and ends with a *, _, #, ', or " (with no preceding characters).
```

Comment:
Is `#` supposed to be here? I thought `#` was used for headings like so `# heading 1`, ie, with a `#` before the text but without a `#` after the text

Reply:
@eVoloshchak it can "start" with `*` in the sense that the mention can be within two `*`, so it'll still technically start with an `@`, but this is saying it can start with `*` becasue it can be like `let's ask *@carlos@expensify.com*`.

This is why I asked about `#`, cause that one is only a leading character unlike `*`, `_`, `'`, `"`, and `~`.

Incidentally @tienifr, the `~` is missing from the comment  `starts and ends with a *, _, #, ', or "...`